src/features/profil/pages/instances/components/ChangeCommittee.tsx

```diff
@@ -9,9 +9,9 @@ import VoxCard from '@/components/VoxCard/VoxCard'
 import { useGetCommittees, useSetMyCommittee } from '@/services/committee/hook'
 import { Diamond, X } from '@tamagui/lucide-icons'
 import { useMedia, YStack } from 'tamagui'
+import { InstanceCardHeader } from '../../../../../components/InstanceCard/InstanceCard'
```

Comment:
qu'en penses tu de passer par l'alias `@` à la place du relative (pareil pour tous les autres import `../..`) ? 

```suggestion
import { InstanceCardHeader } from '@/components/InstanceCard/InstanceCard'
```

Reply:
j'ai remplacé c'était un déplacement automatique de webstorm :)

Reply:
d'acc, il reste encore des imports relatifs dans les autres fichiers de la PR, tu peux check stp ?

Reply:
Done